recipes/glaze/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "3.2.1":
+    url: "https://github.com/stephenberry/glaze/archive/v3.2.0.tar.gz"
```

Comment:
This is tagged as 3.2.1 but the tar is named 3.2.0, is this a typo?

Reply:
@AbrilRBS 
Sorry. It's my fault.
Fixed.